.github/workflows/release.yml

```
@@ -193,14 +193,61 @@ jobs:
+              --version "$VERSION" \
+              --prev-version "$PREV_VERSION" \
+              --output release_notes.md || {
+              
+              # Last resort: basic release notes
+              echo "⚠️ AI generation failed, creating basic release notes..."
+              cat > release_notes.md << EOF
+# Release v$VERSION
+
+## Changes
+
+$(git log v$PREV_VERSION..HEAD --oneline --no-merges | head -20)
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 232: could not find expected ':'

(syntax-check)